src/Cache.php

```diff
@@ -83,7 +83,7 @@ public function removeLegacyTags(array $data): array
         }
 
         foreach ($symfonySymfony as $version => $composerJson) {
-            if (null !== $alias = $composerJson['extra']['branch-alias'][$version] ?? null) {
+            if (null !== $alias = $composerJson['extra']['branch-alias'][$version] ?? $composerJson['extra']['branch-version'] ?? null) {
```

Comment:
should we actually use `branch-version` for tagged releases ?

Reply:
You're right, I removed these, no need to do anything special for "branch-version"